tests/foreman/cli/test_contentviews.py

```diff
@@ -983,16 +902,22 @@ def test_cv_promote_custom_content(self):
             self.fail(err)
 
         # Associate repo to CV
-        result = ContentView.add_repository({u'id': new_cv['id'],
-                                             u'repository-id': new_repo['id']})
-        self.assertEqual(result.return_code, 0,
-                         "Repo was not associated to selected CV")
+        result = ContentView.add_repository({
+            u'id': new_cv['id'],
+            u'repository-id': new_repo['id'],
+        })
+        self.assertEqual(
+            result.return_code, 0,
+            "Repo was not associated to selected CV"
```

Comment:
You have removed the third parameter, the error message that gets displayed to the user, in other parts of the code, so perhaps you should remove this one here too?